plugins/move_amount.py

```diff
@@ -0,0 +1,22 @@
+import sublime
+import sublime_plugin
+
+# Related reading:
+#     https://forum.sublimetext.com/t/command-to-move-by-several-lines/28508
+#     https://forum.sublimetext.com/t/command-to-move-by-several-lines/28508
```

Comment:
related reading links are the same, should there be two different links or just the one?